guides/common/modules/proc_using-container-registries.adoc

```diff
@@ -30,3 +33,39 @@ Pulling container images:
 ----
 # podman pull {foreman-example-com}/my-image:<optional_tag>
 ----
+
+Pushing container images to the {Project} container registry:
+
+* To indicate which organization, product, and repository the container image belongs to, use container namespaces.
+* All labels are composed of entirely lowercase characters regardless of actual label capitalization.
+* Pushes follow one of two formats:
+[options="nowrap", subs="+quotes,attributes"]
+----
+# podman push <image> example.com/<organization_label>/<product_label>/<repository_name>[:tag]
```

Comment:
You should use _italics_ for example values that are to be replaced by users. I supposed that would look like this:
```suggestion
# podman push _image_ _example.com_/_organization_label_/_product_label_/_repository_name_[:tag]
```

Reply:
Actually, the existing example commands in this module use {foreman-example-com}. Which would translate to:

```suggestion
# podman push _image_ _{foreman-example-com}_/_organization_label_/_product_label_/_repository_name_[:tag]
```

Reply:
Please apply the italics and the attribute for example.com to your other command examples too.